space-cli/cmd/modules/addons/database.go

```diff
@@ -265,6 +273,15 @@ func keepSettingConfig(token, dbType string, account *model.Account, v *model.Sp
 }
 
 func removeDatabase(alias string) error {
+	clusterName := viper.GetString("cluster-id")
+	project := viper.GetString("project")
+	autoRemove := viper.GetBool("auto-remove")
+
+	// change selected account according to cluster name provided
+	if err := utils.ChangeSelectedAccount(clusterName); err != nil {
+		return utils.LogError("Could not set selected account ", err)
+	}
```

Comment:
We can take the clusterName from the selected account

Reply:
the same problem as adding database, btw this cluster-name flag is optional.